examples/ts/btc/omni/index.ts

```diff
@@ -0,0 +1,142 @@
+/**
+ * Send or create an omni asset from a multi-sig wallet at BitGo.
+ *
+ * Copyright 2024, BitGo, Inc.  All Rights Reserved.
+ */
+import { Wallet } from 'modules/bitgo/src';
+import { omniConfig } from './config';
+import * as superagent from 'superagent';
+
+const RECEIVE_ADDRESS = '';
+const SEND_ADDRESS = '';
+const NETWORK = omniConfig.coin === 'tbtc4/' ? 'testnet4' : '';
+const AMOUNT = 1234;
+const ASSET_ID = 31;
+
+async function getWallet() {
+  return await omniConfig.sdk.coin(omniConfig.coin).wallets().get({ id: omniConfig.walletId });
+}
+
+function strToHex(s: string) {
+  return (
+    s
+      .split('')
+      .map((c) => c.charCodeAt(0).toString(16))
+      .join('') + '0'
+  );
+}
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+async function mintOmniAsset(wallet: Wallet, address: string, feeRate = 20_000) {
```

Comment:
I would move this function into a different file called `debugging.ts` so it is separate from the rest of it